src/main/java/com/hm/achievement/command/AbstractRankingCommand.java

```diff
@@ -60,10 +61,8 @@ public void extractConfigurationParameters() {
 
 		langPeriodAchievement = plugin.getChatHeader()
 				+ plugin.getPluginLang().getString(languageHeaderKey, defaultHeaderMessage);
```

Comment:
Why not add the three possible `languageHeaderKey` values to _CmdLang_ and make the constructor of this class take a _CmdLang_ enum value?

Reply:
Probably good idea, sloppy me decided to not look how these variables were defined.